nixos/modules/services/matrix/hookshot.nix

```
@@ -0,0 +1,127 @@
+{
+  config,
+  options,
+  pkgs,
+  lib,
+  ...
+}:
+let
+  cfg = config.services.matrix-hookshot;
+  opt = options.services.matrix-hookshot;
```

This doesn't seem to be used anywhere, so it can probably be dropped

```suggestion
  pkgs,
  lib,
  ...
}:
let
  cfg = config.services.matrix-hookshot;
```